packages/web-components/components/selfie/src/SelfieCaptureScreens.js

```diff
@@ -9,10 +9,24 @@ const COMPONENTS_VERSION = packageJson.version;
 
 async function getPermissions(captureScreen, facingMode = 'user') {
   try {
-    await SmartCamera.getMedia({
+    const stream = await SmartCamera.getMedia({
```

Comment:
we allow switching of camera for selfie capture.

[removeAttribute](https://github.com/smileidentity/web-client/blob/3835618f33c01147b5da632abe78e936e8b9dfb9/packages/web-components/components/selfie/src/selfie-capture/SelfieCapture.js#L552)

you might want to get the camera name after the switch here